Marlin/Configuration.h

```diff
@@ -1252,10 +1258,21 @@
 //#define MESH_BED_LEVELING
 
 /**
- * Normally G28 leaves leveling disabled on completion. Enable
- * this option to have G28 restore the prior leveling state.
+ * Normally G28 leaves leveling disabled on completion. Enable one of
+ * these options to restore the prior leveling state or to always enable
+ * leveling immediately after G28.
  */
 //#define RESTORE_LEVELING_AFTER_G28
+//#define ENABLE_LEVELING_AFTER_G28
```

Comment:
What is the difference between "restore prior leveling state" and "enable leveling"?

Reply:
`RESTORE` turns on leveling after `G28` only if it was on before `G28`.